### PR TITLE
Update the zip crates version to use the latest

### PR DESCRIPTION
**Pull request by sylvestre**
Not sure it will work

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -36,7 +36,7 @@ walkdir = "^2.2"
 num_cpus = "^1.8"
 semver = "^0.9"
 md5 = "^0.6"
-zip = { version = "^0.4", features = ["deflate-zlib"], default-features = false }
+zip = "^0.5"
```
Comment (marco-c) [inline Cargo.toml]: I used the features thing so it would reduce the size of the resulting binary.

Comment (marco-c) [reply Cargo.toml]: If the features didn't change with 0.5, it'd be nice to keep that.

**Comment by marco-c** [comment]
You'll also need to run cargo update to update the Cargo.lock file

File: Cargo.toml (hunk 1 above)
Comment (sylvestre) [reply Cargo.toml]: I tried but it doesn't seem that it is possible to do that anymore?!

Comment (marco-c) [reply Cargo.toml]: Ah it looks like the feature name changed from `deflate-zlib` to `deflate`.

Comment (calixteman) [reply Cargo.toml]: As far as I remember, there were some performance issues with the lastest zip lib.
So we need to be sure that we don't have a too big perf regression (if one of course).

**Comment by sylvestre** [comment]
@marco-c @calixteman updated!

**Comment by marco-c** [comment]
It looks like something must be changed for the new version:
```
UnsupportedArchive("Compression method not supported")
```

**Comment by marco-c** [comment]
Closing in favor of #391.

**Pull request closed without merging**